practice-app/app/views.py

```diff
@@ -66,6 +68,14 @@ class WorldTimeTable(Base, UserMixin):
     languageCode = Column(String(15))
     user_id: Mapped[int] = mapped_column(ForeignKey("User.id"))
 
+class Pokemon(Base):
```

Comment:
Move to another file.

Reply:
Moved to pokemon_api.py. thanks for your suggestion.